### DISCUSSION TITLE
Tried 2 different mail accounts but all sensors are still at 0

### DISCUSSION THREAD

**Discussion by DolfLund** [moralmunky/Home-Assistant-Mail-And-Packages#675] [Q&A]
I've made folders on my imap server on both an office365 account and an icloud account.  The configuration goes fine, and lets me choose my "Package email folder" - but the sensors on both setups remain 0.  There are definitely a few dozen emails in each folder some from as recent as 3hrs ago saying something was delivered. Does the system ignore packages unless it sees the initial confirmation come through?

**Comment by firstof9** [comment]
[This is how it works.](https://github.com/moralmunky/Home-Assistant-Mail-And-Packages/wiki#how-it-works)
